solutions/cses-1091.mdx

```diff
@@ -2,22 +2,27 @@
 id: cses-1091
 source: CSES
 title: Concert Tickets
-author: Danh Ta Chi Thanh
+author: Danh Ta Chi Thanh, Ben Dodge
 ---
 
-## Solution
```

Comment:
Can you reorganize this such that the unofficial editorial link is at the top, followed by the explanation of the solution under a header called "Explanation", and finally the implementation header (with the time complexity) underneath that? It would look something like this:

Unofficial Editorial (C++)
## Explanation
blah blah
## Implementation
**Time Complexity:** O(n+ m)
<Insert code here>